plugin.json

```diff
@@ -16,6 +16,13 @@
     "settings_schema": {
         "header": "To set up the GitLab plugin, you need to register a GitLab OAuth app here https://gitlab.com/profile/applications.",
         "settings": [
+            {
+                "key": "GitlabURL",
+                "display_name": "Gitlab Base URL",
```

Comment:
Maybe `Gitlab URL` instead of `Gitlab Base URL`? 1/5

Reply:
:heavy_check_mark: